frontend/containers/sdgs/component.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react';
+
+import cx from 'classnames';
+
+import Image from 'next/image';
+
+import { SDGS_DATA, SDGS_SIZES } from './constants';
+import { SDGsProps } from './types';
+
+export const SDGs: FC<SDGsProps> = ({ className, size = 'small', sdgs = [] }: SDGsProps) => {
+  return (
+    <div className={cx('flex flex-wrap items-center gap-2', className)}>
+      {sdgs.map(({ id, title }) => {
+        const { image } = SDGS_DATA.find((sdg) => sdg.id === id);
+        return (
+          <span key={id}>
+            <Image
```

Comment:
I wonder if you could add a `title` prop so that the SDG title appears when the mouse hovers the image?

I'm thinking that for Spanish and Portuguese users, the image will probably stay in English, so it'd be nice to have a way for them to see what it represents.